packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx

```diff
@@ -206,7 +209,47 @@ describe('Line chart rendering', () => {
       expect(container).toMatchSnapshot();
     },
     undefined,
-    beforeAll,
+    undefined,
+    !(isTimezoneSet(Timezone.UTC) && isTestEnv()),
+  );
+
+  testWithWait(
```

Comment:
Same goes for these 3 tests as commented above in AreaChart. This also can be refactored to reduce number of code lines.

Reply:
To be fixed in next iteration